src/Modules/Shared/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psm1

```diff
@@ -737,4 +737,18 @@ function ConvertFrom-SddlString
             RawDescriptor = $rawSecurityDescriptor
         }
     }
-}
\ No newline at end of file
+}
+
+function Get-Uptime
+{
+    [CmdletBinding()]
+    [Alias("uptime")]

```

Comment:
The alias collides with the native utility `/usr/bin/uptime` in Linux. I don't think we want PS cmdlets to hide the native utility (e.g. `Invoke-WebRequest` had the alias `curl` and `wget`, and it caused problems in Linux PS), so we should avoid using this alias.